core/templates/pages/topics-and-skills-dashboard-page/topics-list/topics-list.component.html

```diff
@@ -152,6 +276,91 @@
+  <div *ngIf="isSerialChapterLaunchFeatureEnabled()"
+       class="topics-list-mobile e2e-test-mobile-topic-table">
+    <div *ngFor="let topic of topicSummaries; index as idx">
+      <div class="topic-item">
+        <div class="topic-item-header">
+          <div class="topic-item-name-box">
+            <div class="topic-item-name e2e-test-mobile-topic-name">
+              <span>
+                {{ getSerialNumberForTopic(idx) }}.
+              </span>
+              <a [href]="getTopicEditorUrl(topic.id)">
+                {{ topic.name }}
+              </a>
+              <span>({{ topic.getTotalChaptersCounts().length }} stories, {{ topic.subtopicCount }} subtopics, {{ topic.totalSkillCount }} skills)</span>
+            </div>
+            <div class="topic-item-classroom">
+              <span class="topic-item-classroom-header">Classroom: </span>
+              <span *ngIf="topic.classroom">{{ topic.classroom }}</span>
+              <span *ngIf="!topic.classroom">Unassigned</span>
+            </div>
+          </div>
+          <div class="topic-item-options">
+            <button class="fas fa-ellipsis-v topic-edit-box-icon" role="button" aria-label="Edit Options" (click)="changeEditOptions(topic.id)"></button>
+            <div *ngIf="showEditOptions(topic.id)" class="topic-edit-options">
+              <a class="topic-edit-options-edit" [href]="getTopicEditorUrl(topic.id)">
+                <i class="fa fa-pen"></i>
+                <span>Edit</span>
+              </a>
+              <a *ngIf="userCanDeleteTopic" (click)="deleteTopic(topic.id, topic.name)">
+                <i class="fas fa-trash-alt"></i>
+                <span>Delete</span>
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="topic-item-topic-status">
+          <span class="topic-item-topic-status-header">Status</span>
+          <span *ngIf="topic.isPublished && topic.getTotalChaptersCounts().length && topic.getTotalChaptersCounts().length == fullyPublishedStoriesCounts[idx]">
+            Fully Published
+          </span>
+          <span *ngIf="topic.isPublished && (topic.getTotalChaptersCounts().length==0 || topic.getTotalChaptersCounts().length !== fullyPublishedStoriesCounts[idx])">
+            Partially Published
+          </span>
+          <span *ngIf="!topic.isPublished">
+            Not Published
+          </span>
+        </div>
+        <div class="topic-item-topic-description">
+          <span class="topic-item-description-header">Description: </span>
+          <span>{{ topic.description }}</span>
+        </div>
+        <div class="topic-item-entities">
+          <div class="topic-item-topic-entity published-stories-entity">
+            <div class="topic-item-topic-entity-header">
+              Published Stories
+            </div>
+            <span *ngIf="partiallyPublishedStoriesCounts[idx]">
+              {{ partiallyPublishedStoriesCounts[idx] }} Partial
+              <br>
```

Comment:
Don't use br. Use ul/li and style them appropriately to hide the bullets. Ditto elsewhere.

br is just for presentation but in this case this is semantically a list of items so you should use the HTML elements with the correct semantics matching that.

Reply:
Done. I have used div for this, it is also giving the right UI.